tests/org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/highlighting/XtendHighlightingCalculatorTest.java

```diff
@@ -132,11 +140,13 @@ protected XtendClass clazz(String string) throws Exception {
 	}
 	
 	@Test public void testLiteral() {
+		strictMode = true;
 		expectInsignificant(0, 3);
 		expectInsignificant(3, 2);
+		expectTemplateLineBreak(5, 1);
 		expectInsignificant(6, 1);
+		expectTemplateLineBreak(14, 1); //TODO shouldn't this be POSSIBLE_LINE_BREAK
```

Comment:
See org.eclipse.xtend.core.tests.richstring.AbstractRichStringEvaluationTest.testMultiLineTemplate():
This template emits 'foobar  \n' so [14,1] should be template line break, [5,1] is the wrong expectation, though.